NAMESPACE

```diff
@@ -208,7 +208,6 @@ importFrom(rlang,as_string)
 importFrom(rlang,enquo)
 importFrom(rlang,enquos)
 importFrom(rlang,env_clone)
-importFrom(rlang,flatten_if)
```

Comment:
Bah, sorry for not catching the stale namespace my review.

Reply:
@kyleam  Just to clarify: this is what we want, right? ... Drop `flatten_if` / add `list_flatten`? Or something else here?

Reply:
OIC; didn't realize that came after you started reviewing. Thanks.